matter_server/server/device_controller.py

```diff
@@ -962,7 +944,23 @@ def reschedule() -> None:
             )
             # TODO: fix this once OperationalNodeDiscovery is available:
             # https://github.com/project-chip/connectedhomeip/pull/26718
-            reschedule()
+            self._schedule_interview(
+                node_id,
+                min(reschedule_interval + 10, MAX_POLL_INTERVAL),
+            )
+
+    def _schedule_interview(self, node_id: int, delay: int) -> None:
+        """(Re)Schedule interview and/or initial subscription for a node."""
+        assert self.server.loop is not None
+        # pop any existing (re)schedule timer
+        self._sub_retry_timer.pop(node_id, None)
+        self._sub_retry_timer[node_id] = self.server.loop.call_later(
+            delay,
+            asyncio.create_task,
+            self._check_interview_and_subscription(
```

Comment:
We create the coroutine here and then wait delay until the task is created. Instead we should wait delay and then create the coroutine and schedule it as a task. Make an additional callback that creates the task and we can use as target for call_later.

Reply:
Yes, I need more coffee

Reply:
adjusted